### ISSUE TITLE
Boot directly to netloader

### ISSUE THREAD

**Issue by mashers** [mashers/3ds_hb_menu#65]
Hold down Y while booting to boot directly to netlauncher, bypassing loading of other launcher features.

**Comment by ihaveamac** [comment]
sort-of already done since it would check for "Y" when it draws the interface. if you mean not using features such as themes that would make sense.

**Comment by mashers** [comment]
Yes I think the requester (in the GL thread) mean skipping loading other features like themes to get to the netloader more quickly.

**Comment by mashers** [comment]
I'm working on this now. I've got it so it will detect the Y button being held during boot, and this will cause themes, sounds and some other things not to load. There are a few other changes I need to make to get it to display the netloader but this shouldn't be too hard.

Recent commits will include these changes, but will have the keypress detection commented out until the feature is actually working.
